php/libraries/FilesUploadHandler.class.inc

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * An interface containing methods for classes used to handle file
+ * uploads from a client to a server.
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  Main
+ * @author   Nicolas Brossard <nicolasbrossard.mni@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+/**
+ * Handles file uploads from a client to a server. The methods listed in this
```

Comment:
Minor grammatical nits/editing: The first sentence is missing a noun (should be "FilesUploadHandler handles files uploads [...]", and the second sentence has a lot of superfluous wording. It can just be something like "The interface encapsulates the usual things that happen when a set of files is uploaded to a server, namely [...]", though "the usual things" seems a little informal for a class doc, so maybe just: "The interface encapsulates the following behaviours that happen when a set of files is uploaded: [...]" would be even better.

But like I said, that's nitpicking. (Maybe @PapillonMcGill has thoughts on a good way to word it. Her reviews of the spec/README PRs generally had good technical writing suggestions.)

Reply:
I would go with @driusan suggestions: "The interface encapsulates the following behaviours that happen when a set of files is uploaded: [...]" for the second phrase. 

Suggestion for the first phrase is also good or "The FilesUploadHandler interface handles ..." but my first choice would be  "This interface handles ..."